[2024-07-13T14:33:24.591+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-13T14:33:24.604+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: chess_etl.blunders_etl scheduled__2024-06-01T00:00:00+00:00 [queued]>
[2024-07-13T14:33:24.611+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: chess_etl.blunders_etl scheduled__2024-06-01T00:00:00+00:00 [queued]>
[2024-07-13T14:33:24.612+0000] {taskinstance.py:2306} INFO - Starting attempt 2 of 2
[2024-07-13T14:33:24.619+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): blunders_etl> on 2024-06-01 00:00:00+00:00
[2024-07-13T14:33:24.623+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=639) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-07-13T14:33:24.625+0000] {standard_task_runner.py:63} INFO - Started process 641 to run task
[2024-07-13T14:33:24.625+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'chess_etl', 'blunders_etl', 'scheduled__2024-06-01T00:00:00+00:00', '--job-id', '827', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/chess_dag.py', '--cfg-path', '/tmp/tmpfuily8ir']
[2024-07-13T14:33:24.626+0000] {standard_task_runner.py:91} INFO - Job 827: Subtask blunders_etl
[2024-07-13T14:33:24.656+0000] {task_command.py:426} INFO - Running <TaskInstance: chess_etl.blunders_etl scheduled__2024-06-01T00:00:00+00:00 [running]> on host 9bf11a4173a0
[2024-07-13T14:33:24.708+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='chess_etl' AIRFLOW_CTX_TASK_ID='blunders_etl' AIRFLOW_CTX_EXECUTION_DATE='2024-06-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-06-01T00:00:00+00:00'
[2024-07-13T14:33:24.709+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-07-13T14:33:24.710+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/utils/context.py:320: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(k, replacements))

[2024-07-13T14:33:24.716+0000] {base.py:84} INFO - Using connection ID 'chess_project_s3' for task execution.
[2024-07-13T14:37:09.421+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-07-13T14:37:09.422+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-07-13T14:37:09.431+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=chess_etl, task_id=blunders_etl, run_id=scheduled__2024-06-01T00:00:00+00:00, execution_date=20240601T000000, start_date=20240713T143324, end_date=20240713T143709
[2024-07-13T14:37:09.485+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-07-13T14:37:09.506+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-07-13T14:37:09.508+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
